<launch>
  <node pkg="wall_following" type="record_odom_action_server.py" name="record_odom_action_server_node" output="screen"/>
</launch>